pkg/apis/serving/v1alpha2/explainer.go

```diff
@@ -70,11 +70,16 @@ func (e *ExplainerSpec) Validate() error {
 	return nil
 }
 
+// +k8s:openapi-gen=false
 type ExplainerConfig struct {
 	ContainerImage string `json:"image"`
 
-	//TODO add readiness/liveness probe config
+	DefaultImageVersion string `json:"defaultImageVersion"`
```

Comment:
Should this type be owned by explainer.go, or owned by the configmap.go class/package?

Reply:
ye moved to configmap.go